.gitignore

```diff
@@ -1 +1,2 @@
 Manifest.toml
+Dockerfile
```

Comment:
Why is this needed? This seems to be specific to your setup, so it should not go into this project's `.gitignore`.